src/color.rs

```diff
@@ -126,6 +126,145 @@ pub struct Color {
 }
 
 impl Color {
+
+    /// Create a new `Color` from the given RGB values and alpha set to 1.0. 
+    /// The given values must adhere to those laid out in the documentation 
+    /// for [`Color`]. Thus:
+    /// 
+    /// * 0.0 <= red <= 255.0
+    /// * 0.0 <= green <= 255.0
+    /// * 0.0 <= blue <= 255.0
```

Comment:
I'll change them to markup. However I do think that there is value in exposing basic notation methods (especially common ones) to beginner programmers. You have "between x and y" in the module docs, so this could provide a simple way to transition the understanding of someone who otherwise may encounter them with less recourse to obtain understanding (if they have difficulty).